haskell/private/haskell_impl.bzl

```diff
@@ -66,7 +66,11 @@ def _should_inspect_coverage(ctx, hs, is_test):
 
 def _haskell_binary_common_impl(ctx, is_test):
     hs = haskell_context(ctx)
-    dep_info = gather_dep_info(ctx)
+    dep_info = gather_dep_info(ctx, ctx.attr.deps)
```

Comment:
Code duplication here.

Reply:
@guibou What is duplicated here is one list comprehension in two places. See what our [coding guidelines have to say about that](https://github.com/tweag/guides/blob/master/style/Haskell.md#general-style) (that part originally from the GHC coding style document).